conf/settings/estimation/body_to_imu.xml

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE settings SYSTEM "../settings.dtd">
 
-<settings>
+<settings target="ap|nps">
```

Comment:
Right, how about something like https://github.com/flixr/paparazzi/commit/493fde03de35bf68c38302603b7a4809af962d61 to add the possibility to exclude a list of targets with ```target="!foo|bar"```